queries/pony/highlights.scm

```diff
@@ -0,0 +1,289 @@
+; Includes
+
+[
+  "use"
+] @include
+
+; Keywords
+
+[
+  "type"
+  "actor"
+  "class"
+  "primitive"
+  "interface"
+  "trait"
+  "struct"
+  "embed"
+  "let"
+  "var"
+  (compile_intrinsic)
+  "as"
+  "consume"
+  "recover"
+  "object"
+  "where"
+] @keyword
+
+[
+  "fun"
+  "be"
+] @keyword.function
+
+[
+  "in"
+  "is"
+] @keyword.operator
+
+[
+  "return"
+] @keyword.return
+
+; Qualifiers
+
+[
+  "iso"
+  "trn"
+  "ref"
+  "val"
+  "box"
+  "tag"
+  "#read"
+  "#send"
+  "#share"
+  "#alias"
+  "#any"
+] @type.qualifier
+
+; Conditionals
+
+[
+  "if"
+  "ifdef"
+  "iftype"
+  "then"
+  "else"
+  "elseif"
+  "until"
```

Comment:
Should "until" be `@repeat`? What are we using in other languages that have it?

Reply:
Seems like `@repeat` in most languages